### ISSUE TITLE
Feature request: Character industry slots overview

### ISSUE THREAD

**Issue by kwazulueve** [eveseat/seat#268]
What I am looking for is a page where all my characters are listed, grouped per account and with a column in which corp they are.

Right next to that there would be columns: Manufacturing | Research | Reactions | (others?)
Each column shows the following: a number of activily used slots, the maximum number of slots that character has, and the control range in jumps

For example a character with 4 active build jobs, and trained Mass Production V, Advanced Mass Production IV and Supply Chain Management IV would show 4/10/20

This would enable to have a really nice quick overview of which characters have unused build/science/reaction jobs, instead of having to login to them all to check 😄 

I am giving a 250 mil reward to whoever implements this change

**Comment by warlof** [comment]
Hi,

For the first thing, it's what you already get in `All Characters`

Related to all / active industry lines, is something on the dashboard suit you enough ?

**Comment by kwazulueve** [comment]
Yeah the overview of characters exists, but what I wrote was merely the lead-up for the second part

I personally wouldn't do this on the Dashboard, as it would start feel a bit arbitrarily
/character/list or even better a subsection/menu option would seem better to me, there are already very sparse menu options atm
